### DISCUSSION TITLE
What in the world is a Yoshi Engine

### DISCUSSION THREAD

**Discussion by BlueColorsin** [YoshiCrafter29/YoshiCrafterEngine#67] [General]
Why is there like 50 engines now that support softcoded mods like last month there was **2** and now there is like well 50

**Comment by YoshiCrafter29** [comment]
i dont see any other engines that support custom softcoded shaders tho

**Comment by MemTheBun** [comment]
![yoshi engine](https://user-images.githubusercontent.com/92312822/157458987-848fd3b2-50ad-418a-be93-7d6c29db51dc.png)

**Reply by ErenKaya31** [reply]
me: show me the yoshi engine
you or someone else: *shows fnf yoshi engine*
me: I SAID REAL YOSHI ENGINE
you or someone else: *send the picture*

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/97402522/158867593-65dff4aa-7591-458c-9bf4-2382e9119c5d.png)
